common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -80,47 +81,53 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
-                var langTemplate = [
-                    '<div class="grouped-controls">',
-                        '<button class="control toggle-captions" aria-disabled="false">',
-                            '<span class="icon-fallback-img">',
-                                '<span class="icon fa fa-cc" aria-hidden="true"></span>',
-                                '<span class="sr control-text"></span>',
-                            '</span>',
-                        '</button>',
-                        '<button class="control toggle-transcript" aria-disabled="false">',
-                            '<span class="icon-fallback-img">',
-                                '<span class="icon fa fa-quote-left" aria-hidden="true"></span>',
-                                '<span class="sr control-text"></span>',
-                            '</span>',
-                        '</button>',
-                        '<div class="lang menu-container" role="application">',
-                            '<p class="sr instructions" id="lang-instructions"></p>',
-                            '<button class="control language-menu" aria-disabled="false"',
-                                'aria-describedby="lang-instructions" ',
-                                'title="',
+                var langHtml = HtmlUtils.joinHtml(
+                    HtmlUtils.HTML('<div class="grouped-controls">'),
```

Comment:
Yes @clrux, these are the changes I remembered.  I don't remember seeing this before the original PR merged.  I think the code used below, with `HtmlUtils.HTML` used just once in conjunction with `interpolateHtml` and `join` seems much cleaner:
https://github.com/edx/edx-platform/blob/3e9cbabfe4a3acc4c4ee41023f2a0873e7e61dab/common/lib/xmodule/xmodule/js/src/video/09_video_caption.js#L835-L845

Thanks.